cmd/interceptors/main.go

```diff
@@ -72,8 +91,30 @@ func main() {
 	mux.Handle("/", service)
 	mux.HandleFunc("/ready", handler)
 
+	keyFile, certFile, caCert, err := getCerts(ctx, secretLister, kubeClient)
+	if err != nil {
+		return
+	}
+
+	tc, err := triggersclientset.NewForConfig(cfg)
+	if err != nil {
+		return
+	}
+	clusterInterceptorList, err := tc.TriggersV1alpha1().ClusterInterceptors().List(ctx, metav1.ListOptions{})
```

Comment:
This should be in its own function e.g. updateCRDWithCaCert or something like that